sale_order_carrier_auto_assign/models/sale_order.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import models
+from odoo.tools import config
+
+
+class SaleOrder(models.Model):
+
+    _inherit = "sale.order"
+
+    def action_confirm(self):
+        if config["test_enable"] and not self.env.context.get(
```

Comment:
@matiasperalta1 @grindtildeath I don't like such code. Better to add a setting to enable the behaviour that breaks standard one. But not blocking

Reply:
@matiasperalta1 Could you put at least in ROADMAP we need to add a configuration setting ?

Reply:
Why adding this in an other PR ? Would you not prefer to include it here ?

Reply:
I would prefer. But probably @matiasperalta1 did not had opportunity to implement that.